dotnet/sample-apps/aws-sdk/deploy/wrapper/main.tf

```
@@ -33,11 +34,24 @@ module "hello-lambda-function" {
   }
 }
 
+resource "aws_lambda_alias" "provisioned" {
+  name             = "provisioned"
```

```suggestion
  name             = var.name
```

I think that this needs to be something more specific than `provisioned`.  It's not clear to me from the docs whether this is scoped to a particular function or not.

---

I second this and believe it would be a good practice to specify the function name, although it seems to me it not scoped hard to the function name as the functionality would work without it.

---

Done